### ISSUE TITLE
Add ability to crowdsource meaning of arbitrary strings on publishers' websites

### ISSUE THREAD

**Issue by emanuil-tolev** [CottageLabs/OpenArticleGauge#120]
For example on EPMC

``` html
<img usemap="#logo-imagemap" alt="" src="/corehtml/pmc/pmcgifs/logo-nihpa.gif"></img>
```

means it's an author manuscript.

This issue is an idea to allow crowdsourcing the fact that that HTML string means something is an author manuscript. UI would be something like

| input textarea for html string | where in the OAG data model the new field will appear | what value it should take|

e.g.

`| <img usemap="#logo-imagemap" alt="" src="/corehtml/pmc/pmcgifs/logo-nihpa.gif"></img> | licence.provenance.accepted_author_manuscript | True |`

So basically allow the scraper to be configured to look for anything at all and record the fact it was there.

Note this EPMC example will be implemented in simple custom code in the `epmc_license` plugin so initial use case will either be something else, or that custom code should be taken out of epmc_license as this is going in.
